src/main/java/org/ideoholic/curium/model/diary/action/DiaryActionAdapter.java

```diff
@@ -60,4 +58,18 @@ public void deleteRecord() {
         dairyIdsDto.setIdDiary(request.getParameterValues("id"));
         diaryService.deleteRecord(dairyIdsDto);
     }
+    public boolean viewDiaryParent() {
+        DiaryService diaryService = new DiaryService(request,response);
+
+        ViewDiaryParentDto viewDiaryParentDto = new ViewDiaryParentDto();
+        viewDiaryParentDto.setId(request.getParameter("id"));
+        viewDiaryParentDto.setId(request.getParameter("page"));
+
+        DiaryResponseDto diaryResponseDto = diaryService.viewDiaryParent(viewDiaryParentDto);
```

Comment:
Change to: 
`DiaryResponseDto diaryResponseDto = diaryService.viewDiaryParent(viewDiaryParentDto, httpSession.getAttribute(BRANCHID).toString());`